library/private/interactor_impl.h

```diff
@@ -35,15 +35,20 @@ class interactor_impl : public interactor
   interactor_impl(options& options, window_impl& window, scene_impl& scene);
   ~interactor_impl() override;
 
-  interactor& addCommandCallback(
-    std::string action, std::function<bool(const std::vector<std::string>&)> callback) override;
-  interactor& removeCommandCallback(const std::string& action) override;
+  interactor& initializeDefaultCommands() override;
+  interactor& addCommand(const std::string& action,
+    std::function<bool(const std::vector<std::string>&)> callback) override;
+  interactor& removeCommand(const std::string& action) override;
+  std::vector<std::string> getCommandActions() const override;
   bool triggerCommand(std::string_view command) override;
-  interactor& addInteractionCommands(
-    std::string interaction, ModifierKeys modifiers, std::vector<std::string> command) override;
-  interactor& addInteractionCommand(
-    std::string interaction, ModifierKeys modifiers, std::string command) override;
-  interactor& removeInteractionCommands(std::string interaction, ModifierKeys modifiers) override;
+
+  interactor& initializeDefaultBindings() override;
+  interactor& addBinding(const std::string& interaction, ModifierKeys modifiers,
+    std::vector<std::string> commands) override;
+  interactor& addBinding(
+    const std::string& interaction, ModifierKeys modifiers, std::string command) override;
+  interactor& removeBinding(std::string interaction, ModifierKeys modifiers) override;
+  std::vector<std::pair<std::string, ModifierKeys>> getBindingInteractions() const override;
```

Comment:
same here, just `getBindings`?

Reply:
because you dont get access to the commands, just the interactions and modifiers